src/Tizen.Multimedia.Remoting/WebRTC/WebRTCEnums.cs

```diff
@@ -437,4 +437,218 @@ internal enum WebRTCDisplayType
 
         Evas,
     }
-}
\ No newline at end of file
+
+    /// <summary>
+    /// Specifies the type of WebRTC statistics.
+    /// </summary>
+    /// <since_tizen> 10 </since_tizen>
+    [Flags]
+    public enum WebRTCStatisticsCategory
+    {
+        /// <summary>
+        /// Codec.
+        /// </summary>
+        Codec = 1,
+
+        /// <summary>
+        /// Inbound RTP.
+        /// </summary>
+        InboundRtp = 2,
```

Comment:
I meant I think '0x0002' style could be easy to extend further addition when it comes to enum for bitwise operation.